guides/source/active_support_core_extensions.md

```diff
@@ -2994,7 +2994,7 @@ Extensions to `Date`
 
 ### Calculations
 
-NOTE: All the following methods are defined in `active_support/core_ext/date/calculations.rb`.
+NOTE: All the following methods are defined in `active_support/core_ext/date_and_time/calculations.rb`.
```

Comment:
I think in this context the link is correct. If  try read `active_support/core_ext/date/calculations.rb` we can get to know about `DateAndTime::Calculations`. The same things with other refferences in this file: `active_support/core_ext/date_time/calculations.rb`, `active_support/core_ext/time/calculations.rb`

Reply:
It is only right in this context if they methods are indeed defined in `active_support/core_ext/date/calculations.rb` but they are not. I'd remove this note from here and add below each method like it is done with the other methods in this guide.

Reply:
I propose doing the same as those methods here, that as you can see have a note below each method not one note on top of the entire class. https://github.com/belgoros/rails/blob/fbfc8e0beca815ba2b54b488f3009ee83f3a916f/guides/source/active_support_core_extensions.md#to_s-1